server/libs/domain/src/media/media.service.ts

```diff
@@ -93,7 +107,7 @@ export class MediaService {
       return;
     }
 
-    const webpPath = asset.resizePath.replace('jpeg', 'webp');
+    const webpPath = asset.resizePath.replace('jpeg', 'webp').replace('jpg', 'webp');
```

Comment:
If we changed thumbnail extension from `jpeg` to `jpg` in `handleGenerateJpegThumbnail` then it seem we don't need to replace `jpeg` here

Reply:
Maybe. Existing database records will have `jpeg` and I wasn't 100% sure if it was possible for this job to run on some of those paths or not.